www/class/config-generate/servicecategory.class.php

```diff
@@ -0,0 +1,173 @@
+<?php
```

Comment:
This class has many similitudes with hostcategory.class.php. Should be refactored ?